rust/src/datatypes.rs

```diff
@@ -72,14 +75,16 @@ impl Schema {
             .enumerate()
             .find(|&(_,c)| c.name == name)
     }
+}
 
-    pub fn to_string(&self) -> String {
-        let s : Vec<String> = self.columns.iter()
+impl fmt::Display for Schema {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        let s: Vec<String> = self.columns
+            .iter()
             .map(|c| c.to_string())
             .collect();
-        s.join(",")
+        write!(f, "{}", s.join(","))
```

Comment:
Would you like to keep a whitespace after comma? IMHO, it makes it easier to read the schema.

Reply:
I agree, but is there a standard? Ideally we'd test against that